torch/_decomp/decompositions.py

```diff
@@ -765,7 +1010,7 @@ def cudnn_batch_norm(
     )
     # Cudnn return running mean and variance when training is True
     if training:
-        return (a, b, c, input.new_zeros((1,)))
+        return (a, b, c, input.new_zeros((0,), dtype=torch.uint8))
```

Comment:
that's whack

Reply:
Tragically, we’re not very consistent about what we return when a tensor is not needed.

Also see aten.native_batch_norm where I need to return something different for saved_mean/saved_invstd depending on whether it’s cuda or cpu.